docs/tutorial/anonlink-novt.ipynb

```diff
@@ -0,0 +1,942 @@
+      "     -------------------------------------------------------------\n",
+      "\u001b[39m     \u001b[39m2 2 2 2 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 \n",
+      "\u001b[39m     \u001b[39m9 9 9 9 0 0 0 0 0 1 1 1 1 1 2 2 2 2 2 3 3 3 3 3 3 4 4 4 4 4 5 \n",
+      "\u001b[39m     \u001b[39m4 5 7 9 1 3 5 7 9 1 3 4 6 8 0 2 4 6 8 0 2 3 5 7 9 1 3 5 7 9 1 \n",
+      "\u001b[39m     \u001b[39m  . . . . . . . . .   . . . . . . . . .   . . . . . . . . .   \n",
+      "\u001b[39m     \u001b[39m  9 8 7 6 5 4 3 2 1   9 8 7 6 5 4 3 2 1   9 8 7 6 5 4 3 2 1   \n",
+      "\u001b[39m     \u001b[39m                                                              \n",
+      "\n",
+      "------------------------\n",
+      "|       Summary        |\n",
+      "------------------------\n",
+      "|  observations: 5000  |\n",
+      "|min value: 294.000000 |\n",
+      "|  mean : 328.055000   |\n",
+      "|max value: 351.000000 |\n",
+      "------------------------\n"
+     ]
+    }
+   ],
+   "source": [
+    "!anonlink describe 'novt_clk_0.json'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Block dataset\n",
+    "\n",
+    "\n",
+    "Blocking is a technique that makes record linkage scalable. It is achieved by partitioning datasets into groups, called blocks and only comparing records in corresponding blocks. This can reduce the number of comparisons that need to be conducted to find which pairs of records should be linked."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\n",
+      "    \"type\": \"lambda-fold\",\n",
+      "    \"version\": 1,\n",
+      "    \"config\": {\n",
+      "        \"blocking-features\": [1, 2],\n",
+      "        \"Lambda\": 25,\n",
+      "        \"bf-len\": 2048,\n",
+      "        \"num-hash-funcs\": 10,\n",
+      "        \"K\": 30,\n",
+      "        \"input-clks\": true,\n",
+      "        \"random_state\": 0\n",
+      "    }\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open(\"blocking_schema.json\") as f:\n",
```

Comment:
It would be good to give more context. How about: "Similar to the hashing above, the blocking is configured with a schema. For this linkage we chose 'lambda-fold' as blocking technique ..."
Do we already have some docs describing the different blocking techniques? If so, it would be nice to link to it here.